config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'home#index'
+
+  namespace :admin do
```

Comment:
if you have an empty block `do...end` please don't write it
in rare cases, it should be given, but here definitely not

Reply:
removed do...end